corehq/apps/users/forms.py

```diff
@@ -794,13 +799,27 @@ class CommtrackUserForm(forms.Form):
     )
 
     def __init__(self, *args, **kwargs):
+        from corehq.apps.locations.views import base_queryset_for_domain_and_user, get_locations_paylod_from_ids
+
         self.domain = None
+        populate_with = None
         if 'domain' in kwargs:
             self.domain = kwargs['domain']
             del kwargs['domain']
+
+        if 'user' in kwargs:
```

Comment:
That does make sense. Since this form is used for both web and mobile user only and both need the locations pre-populated.